js/main.js

```diff
@@ -12,3 +12,29 @@ function imagePreview(event) {
     image.src = 'images/placeholder-image-square.jpg';
   }
 }
+
+var formInput = document.querySelector('.input-form');
+
+formInput.addEventListener('submit', submitUserInput);
+
+function submitUserInput(event) {
+  event.preventDefault();
+  var dataObject = {};
+  var image = document.querySelector('img');
+  var inputForm = document.querySelector('.input-form');
+  var input = inputForm.elements;
+  for (var i = 0; i < input.length; i++) {
+    if (input[i].name === 'title') {
+      dataObject.title = input[i].value;
+    } else if (input[i].name === 'photo-url') {
+      dataObject.url = input[i].value;
+    } else if (input[i].name === 'notes') {
+      dataObject.notes = input[i].value;
+    }
+  }
```

Comment:
Rather than doing this with a loop (which will include a step for the submit button which will be unnecessary work), I would just set each property directly using the named properties in the `elements` property on the `form` element